src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -175,7 +174,13 @@ public JToken TransformContent(ErrorBuilder errors, JsonSchema schema, FilePath
         {
             var href = GetXrefHref(file, uid, uidCount, obj.Parent == null);
             var monikers = _monikerProvider.GetFileLevelMonikers(errors, file);
-            var xref = new InternalXrefSpec(uid, href, file, monikers, obj.Parent?.Path);
+            var xref = new InternalXrefSpec(
+                uid,
+                href,
+                file,
+                monikers,
+                obj.Parent?.Path,
+                schema == rootSchema ? schema.SchemaType is null ? _documentProvider.GetMime(file).Value : schema.SchemaType : null);
```

Comment:
Use `schema.SchemaType` when it is not root schema.

```suggestion
                schema.SchemaType ?? (schema == rootSchema ? _documentProvider.GetMime(file).Value : null);
```

Reply:
OK.